### PR TITLE
1.0.1 lambda-java-serialization release

### PR DESCRIPTION
**Pull request by smirnoal**
*Description of changes:*
Update Jackson dependencies, junit version, owasp plugin version

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: aws-lambda-java-serialization/RELEASE.CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+### November 21, 2022
+`1.0.1`:
+- Updated Jackson dependencies
```
Comment (msailes) [inline aws-lambda-java-serialization/RELEASE.CHANGELOG.md:3]: Could you include the version number in the change long

**Review by andclt — APPROVED** [review]

**Review by toniStepanofAWS — APPROVED** [review]

File: aws-lambda-java-serialization/RELEASE.CHANGELOG.md (hunk 1 above)
Comment (smirnoal) [reply aws-lambda-java-serialization/RELEASE.CHANGELOG.md]: updated

**Review by msailes — APPROVED** [review]
LTGM

**Comment by jeromevdl** [comment]
With this update, I have the following error :

```
java.io.UncheckedIOException: 
com.amazonaws.lambda.thirdparty.com.fasterxml.jackson.databind.exc.InvalidDefinitionException: Joda date/time type `org.joda.time.DateTime` not supported by default: add Module "com.amazonaws.lambda.thirdparty.com.fasterxml.jackson.datatype:jackson-datatype-joda" to enable handling
 at [Source: (BufferedInputStream); line: 13, column: 22] (through reference chain: com.amazonaws.services.lambda.runtime.events.SNSEvent["Records"]->java.util.ArrayList[0]->com.amazonaws.services.lambda.runtime.events.SNSEvent$SNSRecord["Sns"]->com.amazonaws.services.lambda.runtime.events.SNSEvent$SNS["Timestamp"])
```

When adding the following dependency:
```
        <dependency>
            <groupId>com.fasterxml.jackson.datatype</groupId>
            <artifactId>jackson-datatype-joda</artifactId>
            <version>2.14.0</version>
        </dependency>
```

and adding the JodaModule in com.amazonaws.services.lambda.runtime.serialization.events.LambdaEventSerializers: 
```
factory.getMapper().registerModules(new DateModule(), new DateTimeModule(classLoader), new JodaModule());
```
I have the following one:
```
java.io.UncheckedIOException: 
com.amazonaws.lambda.thirdparty.com.fasterxml.jackson.databind.exc.MismatchedInputException: Cannot construct instance of `org.joda.time.DateTime` (although at least one Creator exists): no String-argument constructor/factory method to deserialize from String value ('2020-10-08T16:06:14.656Z')
 at [Source: (BufferedInputStream); line: 13, column: 22] (through reference chain: com.amazonaws.services.lambda.runtime.events.SNSEvent["Records"]->java.util.ArrayList[0]->com.amazonaws.services.lambda.runtime.events.SNSEvent$SNSRecord["Sns"]->com.amazonaws.services.lambda.runtime.events.SNSEvent$SNS["Timestamp"])
```
Am I missing something?

**Comment by smirnoal** [comment]
thanks for reporting that, do you mind sharing a unit test for this?

**Comment by jeromevdl** [comment]
if you update [aws-lambda-java-tests](https://github.com/aws/aws-lambda-java-libs/tree/master/aws-lambda-java-tests) with this new version and run tests, you'll get it.

**Comment by Watlas** [comment]
While it doesn't come out in the new release, is there a workaround for this problem?

image

**Comment by andclt** [comment]
Hi @Watlas, could you please try updating to the latest version (`1.1.2`) of `aws-lambda-java-serialization`. Ref: https://github.com/aws/aws-lambda-java-libs/pull/410

**Pull request merged**